ui/app/components/send/from-dropdown.js

```diff
@@ -0,0 +1,94 @@
+const Component = require('react').Component
+const h = require('react-hyperscript')
+const inherits = require('util').inherits
+const Identicon = require('../identicon')
+
+module.exports = FromDropdown
+
+inherits(FromDropdown, Component)
+function FromDropdown () {
+  Component.call(this)
+}
+
+FromDropdown.prototype.renderSingleIdentity = function (
+  account, 
+  handleClick, 
+  inList = false, 
```

Comment:
I'd consider handling `inList` differently. This is a function with a boolean variable for control flow...when you use it anywhere else, it adds another layer of abstraction for the reader to figure out.

See L66 for an example, it's hard to tell what `true` means.  What do you thinkn of moving that check out (whenever it gets implemented) into a function?